extensions/BugModal/lib/WebService.pm

```diff
@@ -30,9 +30,9 @@ sub rest_resources {
     return [
         # return all the products accessible by the user.
         # required by new-bug
-        qr{^/bug_modal/products}, {
+        qr{^/bug_modal/initial_loader}, {
```

Comment:
Yes, it is just used by new-bug, and the the data it provides is more than just products, hence the name change.